DESCRIPTION

```diff
@@ -4,24 +4,40 @@ Description: Programmatic interface to the
     Web Service methods provided by the National 'Phenology' Network
     (<https://usanpn.org/>), which includes data on various life history
     events that occur at specific times.
-Version: 0.1.4.9100
+Version: 1.1.1
+Date: 2020-10-27
 License: MIT + file LICENSE
 Authors@R: c(
-    person("Scott", "Chamberlain", role = c("aut", "cre"), email = "myrmecocystus@gmail.com"),
-    person("Lee", "Marsh", role = "aut", email = "npnlee85@gmail.com")
+    person("Lee", "Marsh", role = c("aut", "cre"), email = "lee@usanpn.org"),
+    person("Scott", "Chamberlain", role = "aut", email = "myrmecocystus@gmail.com"),
+    person("Alyssa", "Rosemartin", role= "aut", email = "alyssa@usanpn.org"),
+    person("Kevin", "Wong", role = "aut", email = "kevin@usanpn.org")
     )
 LazyData: yes
 Imports:
     stats,
     httr (>= 1.1.0),
     jsonlite (>= 0.9.19),
-    plyr,
     data.table (>= 1.9.6),
-    tibble
+    sf,
+    tibble,
+    curl,
+    raster,
+    rgdal,
+    XML,
+    sp,
+    tidyverse,
+    magrittr,
+    plyr
 Suggests:
     ggplot2,
     testthat,
-    covr
+    covr,
+    knitr,
+    markdown,
+    rmarkdown,
+    RColorBrewer
+RoxygenNote: 7.1.0
+VignetteBuilder: knitr
 URL: https://docs.ropensci.org/rnpn (website)
```

Comment:
you need a trailing slash in the `https://docs.ropensci.org/rnpn` url - cran will complain about this if not there